### ISSUE TITLE
Morphology should have an order of tags

### ISSUE THREAD

**Issue by Labrus** [constructicon/russian#39]
The tags in Morphology (Advanced search tab) should be ordered in a specific way, preferably with some headers. We will prepare this order and the headers to make this explicit.
I have prepared the order, available at [https://docs.google.com/document/d/1qXOWsVbrq3do-WHGKHd-SNGPFlcOq35V/edit](url) but should be first approved by Laura.

VERSION WITH ABBREVIATIONS, the order of subcategories should be as indicated

Case
- Nom
- Gen
- Gen2
- Dat
- Acc
- Ins
- Loc
- Voc - is not in the database yet but can appear in the future

Gender
- M
- F
- N

Number
- Sg
- Pl

Aspect
- Pfv
- Ipfv

Mood
- Ind
- Imp
- Sbjv
- Inf

Tense
- Past
- Present
- Future

Person
- 1
- 2
- 3

Form
- Short
- Bare
- Cmp

Part of speech
- Adj
- Adv
- Conjunction (heading):
Cconj
Sconj
- Cnv
- Intj
- Noun
- Numeral (heading):
NumColl
NumCrd
NumOrd
- Pred
- Prep
- Pronoun (heading):
PronDem
PronInt
PronPers
PronPoss
- Participle (heading):
PtcpAct
PtcpPass
- Verb 

Syntactic units
- Cl
- Cop
- NP
- PP
- VP
- XP

**Comment by bast** [comment]
This will require some more programming though. A bit similar as in #38. So before you do a lot of work on this, let us talk about it because I am not sure I will have time to implement this extra feature very soon.

**Comment by Labrus** [comment]
I agree. In order to bring this issue forward, the best thing I can do is to provide all data for the Morphology column. So today I have annotated completely the first 916 lines for Morphology. This also showed me that the tags I have put in this list are present in our data. In the search window the tags are fed from the database, so eventually the list in the window on the website will be longer. I will make my best to complete this annotation as soon as I can.

**Comment by Labrus** [comment]
All 2267 lines are annotated now for Morphology, so the data is at place.

**Comment by Labrus** [comment]
Laura has approved the order, groupings and headings of Morphology tags. Above I have updated the initial version and added the version with full, not abbreviated terms.

**Comment by Labrus** [comment]
I updated the ordering and grouping of Morphology tags as a result of more discussions within our group and also meeting with Laura. Now it is final.

**Comment by Labrus** [comment]
If programming the order of tags in search window is problematic and too time consuming in the current situation, please let me know and we will instead find the solution how to represent this order in the Instructions page instead. The same for Semantic types #38.

**Comment by Labrus** [comment]
We suggest that we keep alphabetical order (as it is now) in search window in #38 and #39. In order to represent the system of tags and how they are organised (the order we suggested above), we will add a section on Morphology and a section on Semantic types in the Instructions. This issue is resolved.

**Comment by bast** [comment]
Same here: the fact that we may choose not to implement this now does not mean that it should not be implemented later so I would keep this open.
